Docsite: clarify a changelog entry format (#73902)
ansible/ansible: docs/docsite/rst/community/development_process.rst

Each changelog entry must contain a link to its issue between parentheses at the end. If there is no corresponding issue, the entry must contain a link to the PR itself.

Most changelog entries are ``bugfixes`` or ``minor_changes``.

Changelog fragment entry format
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> When writing a changelog entry, follow the format:

Reviewer: ```suggestion
When writing a changelog entry, use the following format:
```

.. code-block:: yaml

>   - scope - description starting with a lowercase letter and ending with a dot (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).

Reviewer 2: ```suggestion
  - scope - description starting with a lowercase letter and ending with a dot at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```

>   - scope - description starting with a lowercase letter and ending with a dot at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).

Reviewer: ```suggestion
  - scope - description starting with a lowercase letter and ending with a period at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```

Author: didn't know that the "period" is more suitable, thanks:)
committed

> The scope is usually a module or plugin name or group of modules or plugins, for example, ``lookup plugins``.

Reviewer 2: ```suggestion
The scope is usually a module or plugin name or group of modules or plugins, for example, ``lookup plugins``. While module names can (and should) be mentioned directly (``foo_module``), plugin names should always be followed by the type (``foo inventory plugin``).
```

> For changes that are not really scoped (for example, which affect a whole collection), follow the format:

Reviewer: ```suggestion
For changes that are not really scoped (for example, which affect a whole collection), use the following format:
```

.. code-block:: yaml

>   - Description starting with an uppercase letter and ending with a dot (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).

Reviewer 2: ```suggestion
  - Description starting with an uppercase letter and ending with a dot at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).
```
